collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDownloader.java

```diff
@@ -63,8 +65,8 @@ public void downloadForm(ServerFormDetails form, @Nullable ProgressReporter prog
                 formsRepository.restore(formOnDevice.getDbId());
             }
         } else {
-            List<Form> allSameFormIdVersion = formsRepository.getAllByFormIdAndVersion(form.getFormId(), form.getFormVersion());
-            if (!allSameFormIdVersion.isEmpty() && !form.getDownloadUrl().contains("/draft.xml")) {
+            preExistingFormsWithSameIdAndVersion = formsRepository.getAllByFormIdAndVersion(form.getFormId(), form.getFormVersion());
+            if (!preExistingFormsWithSameIdAndVersion.isEmpty() && !form.getDownloadUrl().contains("/draft.xml")) {
                 analytics.logEventWithParam(DOWNLOAD_SAME_FORMID_VERSION_DIFFERENT_HASH, "form", AnalyticsUtils.getFormHash(form.getFormId(), form.getFormName()));
```

Comment:
I think this log doesn't make much sense now. I mean it would be good to know if it happens for central users somehow (we can't rule out that there is a bug in central too) but here we don't have that knowledge. We log the event only if it's not central draft but it might very well be any other compatible server like ONA where replacing forms and adding new ones with the same `id` and `version` is perfectly fine and we shouldn't care. Should I just get rid of it or you want to try to determine if it's central? @lognaturel @seadowg

Reply:
I've been thinking about this and can't come up with a use for the analytics. So yes, please remove.